### PR TITLE
871723: Added circular chart 3d features sample in TypeScript and JavaScript.

### PR DESCRIPTION
**Pull request by Soundeswari-Selvam**
**Description:**

Added circular chart 3d features samples.

### CODE REVIEW

**Comment by Soundeswari-Selvam** [comment]
rebuild

File: ej2-javascript/circular-chart-3d/tool-tip.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+platform: ej2-javascript
+control: Tooltip 
+publishingplatform: ##Platform_Name##
+documentation: ug
+domainurl: ##DomainURL##
+---
+
+# Tooltip in ##Platform_Name## Circular Chart 3D control
+
+Tooltip for the Circular Chart 3D can be enabled by using the `enable` property.
```
Comment (Gowrimathi) [inline ej2-javascript/circular-chart-3d/tool-tip.md:14]: by using enable property in ?

**Review by Gowrimathi — CHANGES_REQUESTED** [review]
Requested changes

**Review by Gowrimathi — APPROVED** [review]
Fine

**Pull request merged**